tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
-- name: Include OS-specific variables.
-  include_vars: "{{ ansible_os_family }}.yml"
+- name: set ntp_daemon to 'ntp' as default
```

Comment:
I must admit that @geerlingguy initial approach here is more clear and readable. In fact, pretty commonnly used and expected technique.